fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustRatingMode.tsx

```diff
@@ -41,6 +57,8 @@ export class N1Limit extends DisplayComponent<{
 
     this.N1LimitType.setConsumer(sub.on('thrust_limit_type').whenChanged());
     this.N1ThrustLimit.setConsumer(sub.on('thrust_limit').whenChanged());
+    this.N1ThrustLimitIdle.setConsumer(sub.on('thrust_limit_idle').whenChanged());
+    this.N1ThrustLimitToga.setConsumer(sub.on('thrust_limit_toga').whenChanged());
```

Comment:
```suggestion
    this.N1ThrustLimitIdle.setConsumer(sub.on('thrust_limit_idle'));
    this.N1ThrustLimitToga.setConsumer(sub.on('thrust_limit_toga'));
```
.whenChanged is redundant. The consumer already checks this in it's equality comparison.

Reply:
Will be changed.